libcudacxx/include/cuda/std/__concepts/concept_macros.h

```diff
@@ -316,29 +356,6 @@
 
 #  endif
 
-////////////////////////////////////////////////////////////////////////////////
-// _LIBCUDACXX_TEMPLATE
-// Usage:
-//   _LIBCUDACXX_TEMPLATE(typename A, typename _Bp)
-//     _LIBCUDACXX_REQUIRES( Concept1<A> _LIBCUDACXX_AND Concept2<_Bp>)
-//   void foo(A a, _Bp b)
-//   {}
-#  if (defined(__cpp_concepts) && _CCCL_STD_VER >= 2020)
-#    define _LIBCUDACXX_TEMPLATE(...) template <__VA_ARGS__>
-#    define _LIBCUDACXX_REQUIRES(...) requires __VA_ARGS__
-#    define _LIBCUDACXX_AND           &&
-#    define _LIBCUDACXX_TRAILING_REQUIRES(...) \
-      ->__VA_ARGS__                            \
-        requires _LIBCUDACXX_PP_EXPAND
-#  else
-#    define _LIBCUDACXX_TEMPLATE(...) template <__VA_ARGS__
-#    define _LIBCUDACXX_REQUIRES(...) \
-      , bool _LIBCUDACXX_true_ = true, _Concept::_Enable_if_t < __VA_ARGS__ && _LIBCUDACXX_true_, int > = 0 > /**/
-#    define _LIBCUDACXX_AND                         &&_LIBCUDACXX_true_, int > = 0, _Concept::_Enable_if_t <
-#    define _LIBCUDACXX_TRAILING_REQUIRES_AUX_(...) , __VA_ARGS__ >
-#    define _LIBCUDACXX_TRAILING_REQUIRES(...)      ->_Concept::_Requires_t < __VA_ARGS__ _LIBCUDACXX_TRAILING_REQUIRES_AUX_
-#  endif
-
 ////////////////////////////////////////////////////////////////////////////////
 // _LIBCUDACXX_REQUIRES_EXPR
```

Comment:
i think you should go all the way and change the other `_LIBCUDACXX_*` concepts macros. leave no `_LIBCUDACXX` macro in this file.

Reply:
I want to do that in a follow up, because I also want to try and clean that implementation up a bit

Reply:
i'd rather the cleanup in its own commit. i don't want to have to change half the macro names in my prs, and then do it again for the other half. i want to do it all once.

Reply:
i fixed the merge conflicts and renamed the rest of the `_LIBCUDACXX*` concepts macros.

Reply:
Renamed everything